### ISSUE TITLE
Game trying to install dotnet even though it's installed and failing

### ISSUE THREAD

**Issue by amiroo54** [bluemagic123/tModLoader#4274]
### Version

Latest 1.4.4-stable

### OS

Linux

### Platform

Steam

### This bug affects...

Gameplay as a Player

### Description

I click launch on tmodloader, Steam says it's running but no window opens. 4 dotnet-install bash scripts processes are there but they dosn't seem to be doing anything. I have installed the dotnet-sdk and dotnet-runtime through my package manager (pacman).
One thing of note, when I reinstall the dotnet it launchers once but every time after it just goes back to the same thing. 

### Log File

[Launch.log](https://github.com/user-attachments/files/16011889/Launch.log)

### Steps to reproduce

1. Install Terraria and TmodLoader
2. Open and close TmodLoader
3. Open the TmodLoader again.

### Expected Behavior

The game launching normally using the already installed dotnet.

### Actual Behavior

The game not launching and trying to install dotnet.

### Reproduction frequency

Every time for me, but for other machines might not happen at all.

### Additional Information

This happens both on stable version and preview-v2025.05

**Comment by JavidPack** [comment]
If you are seeing success after installing dotnet manually but only once, then it's possible dotnet isn't ending up on the PATH permanently. You can do that by editing bash_profile. I don't know if the pacman install does that for you, you can open a new terminal and run `dotnet --info` to see if it is on PATH.

I'm not making sense of that log, not sure why the launch script install is failing, it may be possible to edit the launch scripts to not attempt to install dotnet and just run `dotnet tModLoader.dll` instead if they keep failing.

**Comment by JavidPack** [comment]
This issue has either no recent activity or no reproducible steps, so it is being closed. We assume this issue has solved itself in some manner. Feel free to open a new issue if the issue persists and fill out the issue template completely.
